org/pr/android-installable-build.ts

```diff
@@ -0,0 +1,100 @@
+import { danger } from "danger"
+import { Status } from "github-webhook-event-types"
+import fetch from "node-fetch"
+import * as url from "url";
+import * as path from "path";
+
+const CIRCLECI_TOKEN: string = process.env['CIRCLECI_TOKEN']
+const PERIL_BOT_USER_ID: number = parseInt(process.env['PERIL_BOT_USER_ID'], 10)
+
+async function circleCIArtifacts(owner: string, repo: string, buildNumber: number) {
+    const url = `https://circleci.com/api/v1.1/project/gh/${owner}/${repo}/${buildNumber}/artifacts?circle-token=${CIRCLECI_TOKEN}`
+    const res = await fetch(url)
+    if (res.ok) {
+      return res.json()
+    }
+    return []
+}
+
+export default async (status: Status) => {
+    if (status.state !== "success") {
+      return console.log(
+        `Not a successful state - got ${status.state}`
+      )
+    }
+
+    if (status.context !== "ci/circleci: Installable Build") {
```

Comment:
Does this mean that we're going to call all the builds that have an apk "Installable Build"?

Reply:
Yes. The `context` string in the webhook is the only way for us to know what type of build this is.

Alternatively, we could check the artifacts of every CircleCI build to check if there is an APK and post a comment with that.

Reply:
👍 Perfect! Thanks for clarifying. 
Makes sense to me.